docs-2.0/nebula-dashboard-ent/4.cluster-operator/6.settings.md

```diff
@@ -0,0 +1,21 @@
+# Other settings
+
+This topic will describe other settings in Nebula Dashboard.
+
+- Information: It shows cluster name, creates time, and creator.
```

Comment:
It --》it

Reply:
please reorganize [cluster name, creates time, and creator.]

Reply:
```suggestion
- Information: shows the cluster name, the creation time, and the creator.
```